velox/vector/arrow/tests/ArrowBridgeSchemaTest.cpp

```diff
@@ -123,6 +123,9 @@ TEST_F(ArrowBridgeSchemaExportTest, scalar) {
 
   testScalarType(TIMESTAMP(), "ttn");
   testScalarType(DATE(), "tdD");
+
+  testScalarType(DECIMAL(10, 4), "d:10,4");
```

Comment:
Can you add the type to `ArrowBridgeSchemaTest.validateInArrow` below?  Here it only test the round trip, the test below will actually make sure Arrow can read it.